### DISCUSSION TITLE
Getting Started documentation to be fixed

### DISCUSSION THREAD

**Discussion by rjshah** [dexidp/dex#2148] [Q&A]
I am trying to install dex, following steps mentioned on https://dexidp.io/docs/getting-started/
Pointers to example config takes to a yaml file which is marked deprecated. Please update the getting-started page appropriately to help install Dex quickly.

**Comment by nabokihms** [comment]
Hello @rjshah. 
1. There is a link to appropriate config files just in the header of the deprecated configuration file. We need time (or help) to update all the links.
2. Deprecated configuration example is still valid and has all the information you need to set up Dex.

Hope this helps 🙌
